etc/travis/suites/application/before_script.sh

```diff
@@ -9,4 +9,4 @@ run_command "bin/console doctrine:migrations:migrate --no-interaction --env=test
 
 print_header "Setting the web assets up" "sylius"
 run_command "bin/console assets:install --env=test_cached --no-debug -vvv" || exit $?
-run_command "npm run gulp" || exit $?
+run_command "npm run yarn run gulp" || exit $?
```

Comment:
Hmm I'm not sure, what if you install it locally (not `-g`) and call `./node_modules/.bin/yarn`?

Also, did you see [this](https://github.com/travis-ci/travis-ci/issues/6720#issuecomment-254988120)?